xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/webjar/dropdown-menu.html

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div class="livedata-dropdown btn-group">
+
+    <a class="btn btn-default dropdown-toggle" title="More Actions" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true" role="button">
+      <span class="fa fa-ellipsis-v"></span>
+    </a>
+
+    <ul class="dropdown-menu">
+
+      <li class="dropdown-header">Change Layout</li>
+      <li
+        v-for="layout in data.query.layouts"
+      >
+        <a href="#" @click.prevent="logic.loadLayout(layout)">
+          <span :class="data.meta.layoutDescriptors[layout].icon"></span>
+          {{ data.meta.layoutDescriptors[layout].name }}
+        </a>
+      </li>
+
+    </ul>
+
+  </div>
+</template>
+
+
+<script>
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+define(["Vue"], function (Vue) {
+  Vue.component("dropdown-menu", {
+
+    name: "dropdown-menu",
+
+    template: template,
+
+    props: {
+      logic: Object,
+    },
+
+    computed: {
+      data: function () { return this.logic.data; },
+    },
+
+  });
+});
+</script>
+
+
+<style>
+
+.livedata-dropdown .btn-default {
+  background-color: #f8f8f8;
+  background-image: none;
+  border-color: #e5e5e5;
+  box-shadow: none;
+  color: #333333;
+  text-shadow: none;
+}
```

Comment:
Why do you need this custom styles? I'm worried because this doesn't respect the configured color theme.

Reply:
I was trying to get the same style of the existing dropdown menu on the page, because the default one (from bootstrap) looks strange and not coherent with the page design. But yes, it should be better to use scss for this, I'm working on it!

Reply:
Need to find a solution to be able to use LESS effectively for vue component. As there are higher priority issues, it is delay for now.